packages/node_modules/@ciscospark/widget-recents/src/components/RecentsHeader.css

```diff
@@ -39,3 +39,8 @@
   min-width: 40px;
   height: 40px;
 }
+
+/* To remove default native browser blue outline on button focus state*/
```

Comment:
This was requested by the collab-ui team. We pressed back on it, but they insisted, so it needs to stay.

Reply:
according to the mockups and web client, there is no blue outline on focus state.